extensions/ql-vscode/src/common/errors.ts

```diff
@@ -22,6 +22,14 @@ export class RedactableError extends Error {
       .join("");
   }
 
+  public get fullMessageWithStack(): string {
+    if (!this.stack) {
+      return this.fullMessage;
+    }
+
+    return `${this.fullMessage}\n${this.stack}`;
```

Comment:
Should we use `os.EOL` rather than `\n`?

Reply:
Unfortunately we can't use that because the `errors.ts` module is also imported from the webview which doesn't have access to `os.EOL`. I'm not sure whether `\n` works in VS Code's log output, but I don't think it's worth the complexity of trying to determine whether we're in the webview and then deciding to use either `os.EOL` or `\n`.